packages/components/table/src/table-body/styles-helper.ts

```diff
@@ -29,11 +29,10 @@ function useStyles<T>(props: Partial<TableBodyProps<T>>) {
     const classes = [ns.e('row')]
     if (
       parent?.props.highlightCurrentRow &&
-      row === props.store.states.currentRow.value
+      ref(row).value === props.store.states.currentRow.value
```

Comment:
Why should we change the content of the tabel component here?

Reply:
This is a mistake, originated from some misunderstanding of the response, I am very sorry